python/lsst/pipe/tasks/selectImages.py

```diff
@@ -309,11 +311,11 @@ def runDataRef(self, dataRef, coordList, makeDataRefList=True, selectDataList=[]
             scatterSize = sigmaMad(starSize - psfSize)
 
             valid = True
-            if medianE1 > self.config.maxEllipResidual:
+            if self.config.maxEllipResidual and medianE1 > self.config.maxEllipResidual:
                 self.log.info("Removing visit %s because median e residual too large: %f" %
```

Comment:
Were you able to confirm that `medianE1` is the correct comparison variable here?  All the plots I've seen from Takashi Hamana have `medianE` in the label (but that's not a guarantee of what is actually plotted), but I also note that @rearmstr returns `medianE` in his sample script here:
http://jeeves.astro.princeton.edu/pipermail/hsc_software/all/5853.html

I also wonder if it's worth adding the config value used in the log.info messages (i.e. print explicitly what "too large" is)?